### PR TITLE
readme: add running instructions

### PR DESCRIPTION
**Pull request by kopiczko**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,36 +5,99 @@
...
 
-## Create PostgreSQL database
+Start a minikube.
 
-Create the k8s resources.
+```bash
+minikube start --kubernetes-version 'v1.8.0'
+```
+
+Start PostgreSQL instance inside Kubernetes.
```
Comment (rossf7) [inline README.md]: "Start a"

File: README.md [hunk 2]
```diff
@@ -5,36 +5,99 @@
...
+
+```bash
+kubectl apply -f postgresql.yaml
```
Comment (rossf7) [inline README.md]: Needs manifest/

File: README.md [hunk 3]
```diff
@@ -5,36 +5,99 @@
...
+```
+
+## Running Example Solutions
+
+First follow the steps in Setup section.
+
+### Remote Mode
+
+Remote mode means operator runs outside the Kubernetes cluster and connects to
```
Comment (rossf7) [inline README.md]: mean the

File: README.md [hunk 4]
```diff
@@ -5,36 +5,99 @@
...
+    -kubernetes.incluster="false" \
+    -kubernetes.server="https://$(minikube ip):8443" \
+    -kubernetes.crt="$HOME/.minikube/apiserver.crt" \
+    -kubernetes.key="$HOME/.minikube/apiserver.key" \
+    -kubernetes.ca="$HOME/.minikube/ca.crt"
+```
+
+### In-cluster Mode
+
+Alternative, and most likely desired way to run the operator is the in-cluster
```
Comment (rossf7) [inline README.md]: The alternative

File: README.md [hunk 5]
```diff
@@ -5,36 +5,99 @@
...
+mode. In that mode operator connects to the Kubernetes API with credentials
```
Comment (rossf7) [inline README.md]: the operator

**Review by rossf7 — COMMENTED** [review]
Few 🇬🇧 nits otherwise LGTM 👍

**Comment by kopiczko** [comment]
@rossf7 I think I addressed all the comments. PTALA

File: README.md [hunk 6]
```diff
@@ -5,36 +5,99 @@
...
+    -kubernetes.ca="$HOME/.minikube/ca.crt"
+```
+
+### In-cluster Mode
+
+The alternative, and most likely desired way to run the operator is the
+in-cluster mode. In that mode the operator connects to the Kubernetes API with
+credentials injected by the Kubernetes to the Pod.
+
+Only solution1 and solution2 support this mode.
```
Comment (rossf7) [inline README.md]: Shouldn't this be solution 2 & 3?

**Review by rossf7 — APPROVED** [review]
One query otherwise LGTM

File: README.md (hunk 6 above)
Comment (kopiczko) [reply README.md]: Yes, thanks!

**Pull request merged**